frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts

```diff
@@ -59,16 +61,12 @@ const enableDemoPlugin = (enable: boolean) => {
   if (isLocalDevEnvironment) {
     // for local dev env just trigger any change in the return value to activate the toast
     cy.intercept('/api/check-updates*', { plugins: [] }).as('checkUpdates');
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(CHECK_UPDATE_WAIT);
-    cy.wait('@checkUpdates');
+    cy.wait('@checkUpdates', WAIT_OPTIONS);
     cy.log('Running plugin test locally using bridge.');
   } else {
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(CHECK_UPDATE_WAIT);
     cy.log(`Running plugin test on ci using PLUGIN_PULL_SPEC: ${PLUGIN_PULL_SPEC}`);
   }
-  cy.get(warningToast)
+  cy.byTestID(refreshWebConsoleLink, { timeout: CHECK_UPDATE_WAIT })
```

Comment:
@rhamilto we've had some issues in the past with the toast notification firing. Do you think we should be testing for the rendering of the toast notification here?

Reply:
Not sure I follow.  This change is to simply use a better selector for the toast notification and to extend the timeout to 5 minutes so the fixed wait is not required.